### ISSUE TITLE
incompatible with python3.8

### ISSUE THREAD

**Issue by falkTX** [jpcima/faustpp#16]
I am able to run faustpp python variant with python3.10, but fails when using 3.8

```
Traceback (most recent call last):
  File "/usr/bin/faustpp", line 11, in <module>
    load_entry_point('faustpp==0.0.0', 'console_scripts', 'faustpp')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 490, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2854, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2445, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2451, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/faustpp/main.py", line 8, in <module>
    from faustpp.call_faust import FaustVersion, ensure_faust_version, FaustResult, call_faust
  File "/usr/lib/python3/dist-packages/faustpp/call_faust.py", line 71, in <module>
    def call_faust(dspfile: str, faustargs: list[str]) -> FaustResult:
TypeError: 'type' object is not subscriptable
```

**Comment by falkTX** [comment]
Issue seems to be `list[str]` which was introduced in python3.9

**Comment by falkTX** [comment]
After removing that, I got issues with `dict[...]` usage too, then `tuple[...]` as well :(

I need the tool to run on python3.8 as that is the version used by Ubuntu 20.04, which is still very common for CI / automated builds.

Is there a way to "transpile" the code to remove all these type decorators? I understand they are nice when coding and doing static checks, but backwards compatibility is quite important too...

**Comment by jpcima** [comment]
The `typing` library possibly has compatible equivalents.

eg.  https://docs.python.org/3/library/typing.html#typing.List
(marked as Deprecated since version 3.9)

Can you check these?

**Comment by jpcima** [comment]
Implemented in 3d0e42d, please check it.

**Comment by falkTX** [comment]
can confirm issue is fixed now, was able to trigger a build with stock faustpp from that commit under ubuntu 18.04.
thanks!
